### DISCUSSION TITLE
Enquiry

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#52] [Outreachy 2022] [answered]
@BatoolMM Goodday, i would like to find out how to reachout to you , so you can help review my final application and timeline for the MOOC open educational resources website that i would like to contribute on , during the internship stage......would also like to find out , if one can record a contribution, that hasn't been merged or reviewed

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi, feel free to record a contribution that wasn't merged or reviewed.

In terms of the timeline, feel free to submit your your own timeline, this will not be taken into account during the selection process in OSCSA.

If you have any question, feel free to ask here!

**Comment by jesutobi** [comment]
@BatoolMM  thank you
